packages/prebuild-config/src/plugins/__tests__/__snapshots__/withDefaultPlugins-test.ts.snap

```diff
@@ -157,7 +157,7 @@ Object {
       "CFBundleExecutable": "$(EXECUTABLE_NAME)",
       "CFBundleIdentifier": "$(PRODUCT_BUNDLE_IDENTIFIER)",
       "CFBundleInfoDictionaryVersion": "6.0",
-      "CFBundleName": "$(PRODUCT_NAME)",
```

Comment:
Maybe we should be changing `PRODUCT_NAME` instead.

Reply:
We use `$(PRODUCT_NAME)` in a lot of permission messages so we probably want to resolve that.

Reply:
an aside - i think in most cases we shouldn't use `PRODUCT_NAME` in permission messages - there is only one example of doing this in the ios hig: https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessing-user-data/

![Screen Shot 2021-07-28 at 1 27 40 PM](https://user-images.githubusercontent.com/90494/127391160-3dc5d965-21fe-4619-be62-872ae6bf3d9b.png)

but i agree that changing `PRODUCT_NAME` instead of `CFBundleName` is probably preferable here